### PR TITLE
chore(ci): filter out examples

### PR DESCRIPTION
**Pull request by juliusmarminge**
Continuing on #4237, filter to only install packages and root dependencies on CI

### CODE REVIEW

**Comment by juliusmarminge** [comment]
Don't need to explicitly tell it `--frozen-lockfile` since it's done automatically in CI environments.

**Comment by juliusmarminge** [comment]
Also went ahead and removed all the `yarn` commands etc to keep coherency, added `pnpm-lock.yaml` to `.gitignore`'s so that it's not committed inside the examples.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           registry-url: https://registry.npmjs.org/
           cache: 'pnpm'
       - run: |
-          pnpm install
+          pnpm --filter ./packages/** --filter query install
```
Comment (TkDodo) [inline .github/workflows/ci.yml:28]: is `--filter query` responsible for installing the top level packages?

Comment (juliusmarminge) [reply .github/workflows/ci.yml:28]: Yes, it's the name you have in your root `package.json`

**Review by TkDodo — APPROVED** [review]

**Comment by TkDodo** [comment]
interesting - the publish pipeline now fails with:

```
Run pnpm --filter ./packages/** --filter query install
None of the selected packages has a "./packages/query-broadcast-client-experimental" script

> query@ cipublish /home/runner/work/query/query
> ts-node scripts/publish.ts

sh: 1: ts-node: not found
 ELIFECYCLE  Command failed.
 WARN   Local package.json exists, but node_modules missing, did you mean to install?
Error: Process completed with exit code 1.
```

pretty weird @juliusmarminge ?

**Comment by TkDodo** [comment]
ah, it's missing the quotes!

**Comment by juliusmarminge** [comment]
> ah, it's missing the quotes!

oooh sorry missed the quotes on one of the jobs :/

**Comment by TkDodo** [comment]
fixed: c566c3c1 (and fec3daa3 lol)

**Pull request merged**